.github/workflows/first-interaction.yml

```diff
@@ -0,0 +1,18 @@
+name: Welcome New Contributor
+
+on:
+  pull_request_target:
+    types: [opened]
+  issues:
+    types: [opened]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Congrats on creating your first issue and thanks for time to improve Open Source.'
+        pr-message: 'Congrats on making your first Pull Request and thanks for time to improve Open Source.'
```

Comment:
Please change the message to:

`Congratulations on opening your first pull request to the project! A core maintainer will review your contribution shortly.`